admin_guide/manage_scc.adoc

```diff
@@ -94,7 +93,14 @@ Settings:
   Supplemental Groups Strategy: RunAsAny
     Ranges:				<none>
 ----
-====
+
+Existing `SecurityContext` and `PodSecurityContext` variables could be explained
+by `oc explain`:
+
+----
+$ oc explain Pod.spec.securityContext
+$ oc explain Pod.spec.containers.securityContext
```

Comment:
I don't see how it is related and how it could be helpful in the chapter about SCC. If you want to demonstrate the usage of `oc explain` to inspect SCC common properties, than it should be `oc explain SecurityContextConstraints` but IMHO it has less value to admin.